bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/handler/AstroThingHandler.java

```diff
@@ -272,10 +270,8 @@ private void linkedChannelChange(ChannelUID channelUID, int step) {
      */
     private boolean isPositionalChannelLinked() {
         for (Channel channel : getThing().getChannels()) {
-            if (ArrayUtils.contains(getPositionalChannelIds(), channel.getUID().getId())
-                    && isLinked(channel.getUID().getId())) {
-                return true;
-            }
+            String id = channel.getUID().getId();
+            return (Arrays.asList(getPositionalChannelIds()).contains(id) && isLinked(id));
```

Comment:
This change causes only the first channel to be checked. Is that intentional?

Reply:
You're right ! Well spotted.